### ISSUE TITLE
Reference bug#1541:  MS Word- charts are not detected and read by NVDA.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4909]
**Reported by sumandogra on 2015-02-12 07:54**
MSWord Charts
NVDA does not Inform about the presence of the charts in MS Word documents.  There is no way one can reach the charts and read and analyze the charts in Word documents.

Charts are very important for analyzing the business reports.  Persons who have to study economics also have to read and understand charts a lot.  But as NVDA just cannot read the presence of charts and has no way available to analyze the charts such jobs are not possible with NVDA.

**Comment by siddhartha-iitd** [comment]
[Sample_Charts.docx](https://github.com/nvaccess/nvda/files/77998/Sample_Charts.docx)

**Comment by siddhartha-iitd** [comment]
Sample Charts Document
[Sample_Charts.docx](https://github.com/nvaccess/nvda/files/172073/Sample_Charts.docx)

**Comment by Adriani90** [comment]
This has been solved by #7046. Closing as works for me.
